runtime/filetype.vim

```diff
@@ -1134,7 +1134,7 @@ au BufNewFile,BufRead pf.conf			setf pf
 au BufNewFile,BufRead */etc/pam.conf		setf pamconf
 
 " Pam environment
-au BufNewFile,BufRead .pam_environment		setf pamenv
+au BufNewFile,BufRead pam_env.conf,.pam_environment	setf pamenv
```

Comment:
They are actually the same format for me.
![image](https://user-images.githubusercontent.com/15225902/81583901-389b2000-93dc-11ea-874c-251de8106b00.png)

Maybe your `/etc/security/pam_env.conf` is different.

Reply:
/etc/security/pam_env.conf and ~/.pam_environment share the same format: <https://wiki.archlinux.org/index.php/Environment_variables#Using_pam_env>.

Reply:
Huh!  That's not what the [pam_env manual page](https://manpages.debian.org/buster/libpam-modules/pam_env.7.en.html) says:

> user_envfile=filename
> 
>    Indicate an alternative .pam_environment file to override the default.**The syntax is the same as for /etc/environment**. The filename is relative to the user home directory. This can be useful when different services need different environments.

(emphasis mine)

/etc/environment is documented to use "simple KEY=VALUE" pairs both in this man page, and in the [one for pam_env.conf](https://manpages.debian.org/buster/libpam-modules/pam_env.conf.5.en.html).

While I've been generally impressed by the quality of the ArchWiki, I trust manual pages more.

Meanwhile let me do some experiments on my Ubuntu 20.04 LTS system with ~/.pam_environment variables using different syntaxes, to see which one works when I log in over ssh.

Reply:
There is an issue talk about that: https://github.com/linux-pam/linux-pam/issues/6

An example of .pam_environment: <https://github.com/jonhoo/configs/blob/09563d8dbec005f031482b8ce0bfdef2214af359/env/.pam_environment>.

Reply:
Experiment 1:

```
mg@blynas: ~ $ cat .pam_environment
VAR1=one
VAR2="two"

mg@blynas: ~ $ ssh localhost

mg@blynas:🐌 ~ $ echo $VAR1
one

mg@blynas:🐌 ~ $ echo $VAR2
"two"
```

Experiment 2:

```
mg@blynas: ~ $ cat .pam_environment
VAR3 DEFAULT=three
VAR4 DEFAULT="four"

mg@blynas:🐌 ~ $ echo $VAR3
three

mg@blynas:🐌 ~ $ echo $VAR4
four
```

Looks like **both formats are accepted**, but the KEY=VALUE format doesn't accept quotes.

I would like to throw a chair at the Debian manual pages.